EKF/estimator_interface.h

```diff
@@ -203,6 +204,9 @@ class EstimatorInterface
 	*/
 	virtual bool reset_imu_bias() = 0;
 
+	// return true if the attitude is usable
+	bool attitude_valid() { return ISFINITE(_output_new.quat_nominal(0)) && _control_status.flags.tilt_align && _control_status.flags.yaw_align; }
```

Comment:
Including yaw align in this could be problematic because if the resetMagHeading function fails to complete due to temporary loss of EV data, then all attitude data would be lost.

Reply:
Attitude data can still be usable for stabilisation even when absolute yaw angle is not known. I will remove that condition.